For 003 nrt glad deforestation alerts (#776)
resource-watch/nrt-scripts: for_003_nrt_rw1_glad_deforestation_alerts/README.md

## GLAD Deforestation Alerts Dataset Near Real-time Script
This file describes the near real-time script that retrieves and processes the [the Global Land Analysis and Discovery (GLAD) Alert](http://iopscience.iop.org/article/10.1088/1748-9326/11/3/034008) for [display on Resource Watch](http://resourcewatch.org/data/explore/6ec78a52-3fb2-478f-a02b-abafa5328244).

> The source provided the data as an image collection on Google Earth Engine. Each day of data were split into five images of different regions.

Reviewer: ```suggestion
The source provided the data as an image collection on Google Earth Engine. Each day of data was split into five images of different regions.
```

> For each day, data of five different regions are mosaicked to create a global dataset for display on Resource Watch.

Reviewer: ```suggestion
For each day, the data from the five different regions is mosaicked to create a global dataset for display on Resource Watch.
```
